packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -531,6 +646,21 @@ $Calendar-dayPicker-margin: 10px;
     .monthPicker {
       display: block;
     }
+
+  }
+
+  // Do not hide Month picker if it is the only calendar
+  .pickerOnlyMonthPickerVisible, .pickerIsMonthPickerOverlayVisible{
```

Comment:
The month picker has a smaller width than the day picker, so when changing between the two it doesn't looks right.  In Christina's comps she changed the width of the month picker to match the date picker.  When I made the overlay version I didn't want to change the css of the other month pickers (the side-by-side calendar version), since the css had always been like that and I didn't want to break anything that was out in the wid, so that is why there were so many duplicate styles.  But... I didn't realize that the surface calendar would have another date picker and it would be the side-by-side calendar version.  Since that is the case I do need to update the other calendars and I don't need to duplicate the styles like I did.  I will consolidate.